driver-core/src/main/com/mongodb/client/result/bulk/ClientBulkWriteResult.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.client.result.bulk;
```

Comment:
Suggest: `com.mongodb.client.bulk.result` easier to discover in javadoc. 
Also it help encapsulates this new API

Reply:
The reason I used `com.mongodb.client.result.bulk` is that we already have ``com.mongodb.client.result` with `InsertOneResult`, ..., `DeleteResult`. If we are free not to use that package, then I'd prefer to just put the new result types in `com.mongodb.client.model.bulk`, as I don't find the `result` package useful.

Are you OK with using only `com.mongodb.client.model.bulk`?

Reply:
Yeah sounds great, I want to separate away from the older implementation as much as possible.

Reply:
Done in 3584de5c72b0cf067574146dd36f93712f24f4be.